netmesh/model/netmesh/netmesh.proto

```diff
@@ -21,26 +21,21 @@ syntax = "proto3";
 
 package netmesh;
 
-message Label {
-    map<string ,string> selector = 1;
-}
+import "github.com/ligato/networkservicemesh/pkg/nsm/apis/common/common.proto";
```

Comment:
I see you're doing the client API in pkg/nsm, should everything under netmesh/model/netmesh be moved into the pkg directory as well? If so, I'd do a separate commit with that move rather than combine it here. We can merge this one first.